scripts/cibuild/BuildValidator.ps1

```diff
@@ -19,38 +19,82 @@ param
     [string]$BuildOutputTargetPath,
     [Parameter(Mandatory=$True)]
     [string]$BuildRTM,
-    [switch]$ValidateVsix
+    [Parameter(Mandatory=$True)]
+    [string]$BuildConfiguration,
+    [switch]$ValidateVsix,
+    [switch]$ValidateSigning
 )
 
 
 if ($BuildRTM -eq 'false')
 {   
 
+    $result = 0
     $NuGetClientRoot = $env:BUILD_REPOSITORY_LOCALPATH
-    $NuGetValidator = [System.IO.Path]::Combine($NuGetClientRoot, 'packages', 'NuGetValidator.1.3.7', 'tools', 'NuGetValidator.exe')
-    $VsixLogOutputDir = [System.IO.Path]::Combine($BuildOutputTargetPath, 'LocalizationValidation', 'vsix' )
-    $LocalizationRepository = [System.IO.Path]::Combine($NuGetClientRoot, 'submodules', 'NuGet.Build.Localization', 'localize', 'comments', '15')
+    $NuGetValidator = [System.IO.Path]::Combine($NuGetClientRoot, 'temp', 'NuGetValidator.1.4.0.3', 'tools', 'NuGetValidator.exe')
+    $msbuildExe = 'C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\bin\msbuild.exe'
```

Comment:
hard-coding msbuild location is fine in this case because for VS2017 we are always guranteed that it will be present here and this also guarantees we will get the publicly released RTM version.